web/components/contract/twomba-contract-page.tsx

```diff
@@ -442,13 +453,45 @@ export function TwombaContractPageContent(props: ContractParams) {
             <Row className="my-2 flex-wrap items-center justify-between gap-y-2"></Row>
             {!user && <SidebarSignUpButton className="mb-4 flex md:hidden" />}
             {!!user && (
-              <ContractSharePanel
-                isClosed={isClosed}
-                isCreator={isCreator}
-                showResolver={showResolver}
-                // TODO: upgrade tier
-                contract={props.contract}
-              />
+              <Col>
+                <Row className="items-center gap-2">
```

Comment:
this seems like an awkward place to put the order book, but i'm not sure exactly where it should go

Reply:
agreed but all the other places were worse, it's only there on mobile at least